extensions/json.py

```
@@ -0,0 +1,25 @@
+#
+
+# JSON -> MODULES
+from json import loads
+
+# JSON -> CLASS
+class namespace:
+    # CLASS -> INIT
+    async def init(self, request: Request, response: Response) -> namespace: return self
+    # CLASS -> CONVERT
+    def parse(self, data: dict, keys: list[str]) -> None:
+        for key in [key for key in keys if key in data and isinstance(data[key], str)]: data[key] = loads(data[key])
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)